[RDY] vim-patch:8.0.1832,8.1.0047

janlazo opened the pull request:
**vim-patch:8.0.1832: cannot use :unlet for an environment variable**

Problem:    Cannot use :unlet for an environment variable.
Solution:   Make it work.  Use unsetenv() if available. (Yasuhiro Matsumoto, closes vim/vim#2855)
https://github.com/vim/vim/commit/137374fd6538cf9dee0cb22907728d8fdecb5832

**vim-patch:8.1.0047: no completion for :unlet $VAR**

Problem:    No completion for :unlet $VAR.
Solution:   Add completion. (Jason Franklin)
vim/vim@1983401

janlazo committed:
vim-patch:8.0.1832: cannot use :unlet for an environment variable

Problem:    Cannot use :unlet for an environment variable.
Solution:   Make it work.  Use unsetenv() if available.
            (Yasuhiro Matsumoto, closes vim/vim#2855)
https://github.com/vim/vim/commit/137374fd6538cf9dee0cb22907728d8fdecb5832

k-takata commented:
> Solution: Make it work. Use unsetenv() if available. (Ken Takata,

Just FYI, the author of the patch is actually Yasuhiro Matsumoto, not me.
Bram made a mistake.

janlazo committed:
vim-patch:8.1.0047: no completion for :unlet $VAR

Problem:    No completion for :unlet $VAR.
Solution:   Add completion. (Jason Franklin)
https://github.com/vim/vim/commit/19834010889fc5bfa0f88b3ba83133dae6c0a35d
